conf/application.conf

```diff
@@ -136,6 +136,12 @@ microservice {
         host = localhost
         port = 9871
       }
+      company-information {
+         protocol = https
+         host = api.company-information.service.gov.uk
+         port = 443
+         apiKey = 20757a9d-c286-4fbc-bd7f-3c822bac9517
```

Comment:
Isn't this supposed to be secret? Whose key is this?

Reply:
Right, it needs to be encrypted. It's a developer key